schemas/backend_configuration_schema.json

```diff
@@ -185,6 +185,18 @@
                             "description": "Segment, if indicated, is used to distinguish different subsets of the qubit fabric/chip"
                         }
                     }
+                },
+                "qubit_lo_range": {
```

Comment:
This duplicates the `openpulse_config` values. These values should be removed from `openpulse_config` as they are automatically included from the top-level config.

As these are required properties for openpulse but not for qasm it might make more sense to describe these structures in the top-level properties and reference them as fields. in each of the configs.

Reply:
Addressed in https://github.com/Qiskit/ibm-quantum-schemas/pull/9/commits/16396d9b899f20e78935a1904fc62314a6108514.